roles/patch_host_config/tasks/main.yml

```diff
@@ -50,6 +50,17 @@
       role: "{{ host_role }}"
   when: host_name is defined or host_role is defined
 
+- name: "Create patch body with hostname for {{ host_name }}"
+  set_fact:
+    patch_body:
+      hosts_names:
+        - "{{ host }}"
```

Comment:
_Nitpick warning!_

I know this var will get converted into JSON anyway, but wouldn't the "dict notation" look more JSON-like? I know that this YAML syntax is _technically_ valid for the request body, but it still confused me a bit when I first read it.

```yaml
- name: "Create patch body with hostname for {{ host_name }}"
  set_fact:
    patch_body:
      hosts_names:
        - "{{ host }}"
```
This one seems to me to better indicate what the resulting data type will be:
```yaml
- name: "Create patch body with hostname for {{ host_name }}"
  set_fact:
    patch_body: {
      "hosts_names": [ "{{ host }}" ]
    }
```

Just a thought.